src/cli/lpth.js

```diff
@@ -11,6 +11,9 @@ function parsePath (val) {
   return fs.readFileSync(path.resolve(val))
 }
 
+
+const DEFAULT_ZONE = 'us-east1-b'
```

Comment:
Why change to `DEFAULT_ZONE`? If you need to parse zone out from `docker-compose.yml` without `geth` - you can use [parseConfigFromCommandLine](https://github.com/livepeer/test-harness/blob/master/src/cli/helpers.js#L4) or [parseComposeAndGetAddresses](https://github.com/livepeer/test-harness/blob/master/src/utils/helpers.js#L163) function - it will parse `docker-compose.yml` and will find `zone` on any service (and will put it to `parsedCompose.zone`).